### PR TITLE
New Cult Whetstone Sprite

### PR DESCRIPTION
**Pull request by uraniummeltdown**
Unused: cultw
Used: cultu
Old: oldcultstone

The old eldritch whetstone doesn't even look like a whetstone. New one is animated when it's unused, pulsing red glow
from -tg-
Also changed the whetstone notice spans to a warning

:cl:
imageadd: Adds new sprites to the unused and used eldritch whetstones. Sprites by Fury McFlurry.
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
For some reason this made me want cult soap. Don't know why

**Comment by KingPhilipIII** [comment]
Cult soap when.

File: code/game/gamemodes/cult/cult_items.dm [hunk 1]
```diff
@@ -192,13 +192,15 @@
 /obj/item/weapon/whetstone/cult
 	name = "eldritch whetstone"
 	desc = "A block, empowered by dark magic. Sharp weapons will be enhanced when used on the stone."
-	icon = 'icons/obj/cult.dmi'
-	icon_state = "cultstone"
+	icon_state = "cult_sharpener"
 	used = 0
```
Comment (Jovaniph) [inline code/game/gamemodes/cult/cult_items.dm]: does `used = 0` need to be their as well? It's already '0' in the parent `/obj/item/weapon/whetstone`

Comment (FalseIncarnate) [reply code/game/gamemodes/cult/cult_items.dm]: probably was just copy pasta from the original addition of the subtype

**Comment by tigercat2000** [comment]
👍

**Pull request merged**